lib/utils/location.js

```diff
@@ -45,7 +45,7 @@ export function matchesPageLocation(includes, excludes) {
 /* eslint-disable key-spacing */
 export const regexes = {
 	frontpage:        /^\/(?:hot|new|rising|controversial|top)?(?:\/|$)/i,
-	comments:         /^\/(?:r\/([\w\.]+)\/)?comments(?:\/|$)/i,
+	comments:         /^\/(?:r\/([\w\.]+)\/)?comments(?:\/([a-z0-9]+)\/\w*)?(?:\/|$)/i,
```

Comment:
I know this is a WIP but:

Probably drop the `\/\w*`, since that can be left out or mangled (with any character) and reddit will still accept it.